test/foundry/BlastNativeYield.t.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IOwnableTwoSteps} from "../../contracts/interfaces/IOwnableTwoSteps.sol";
+import {OwnableTwoSteps} from "../../contracts/OwnableTwoSteps.sol";
+import {BlastNativeYield} from "../../contracts/BlastNativeYield.sol";
+import {TestHelpers} from "./utils/TestHelpers.sol";
+import {YieldMode as IBlast__YieldMode, GasMode as IBlast__GasMode} from "../../contracts/interfaces/IBlast.sol";
+
+import {MockBlastPoints} from "../mock/MockBlastPoints.sol";
+import {MockBlastYield} from "../mock/MockBlastYield.sol";
+
+contract BlastNativeYieldOwnableTwoSteps is BlastNativeYield, OwnableTwoSteps {
+    constructor(
+        address _blast,
+        address _blastPoints,
+        address _blastPointsOperator,
+        address _owner
+    ) BlastNativeYield(_blast, _blastPoints, _blastPointsOperator, _owner) OwnableTwoSteps(_owner) {}
+}
+
+contract BlastNativeYieldOwnableTwoSteps_Test is TestHelpers {
```

Comment:
BlastNativeYield_Test is fine

Reply:
the rule of thumb is you want to test BlastNativeYield and BlastNativeYieldOwnableTwoSteps is just a helper contract to help you test BlastNativeYield

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/bc6d3e8172dbe58c7c8bbb4f91fff5b5de6f0aff